CHANGELOG.md

```diff
@@ -2,6 +2,21 @@
 
 All notable changes to this project will be documented in this file.
 
+## Unreleased - as of Sprint 58 end 2017-04-10
+
+### Added
+- Added LenovoXclarity Domain to EMS Events into Automate [(#77)](https://github.com/ManageIQ/manageiq-content/pull/77)
```

Comment:
Please change "Domain" to "Namespace".  I just updated the PR title to be accurate.